### ISSUE TITLE
Clicking on a path sometimes crashes CoinKiller

### ISSUE THREAD

**Issue by mariomadproductions** [ExplosBlue/CoinKiller#59]
See this level for example:

[1-2.sarc.zip](https://github.com/Arisotura/CoinKiller/files/6150087/1-2.sarc.zip)

**Comment by ExplosBlue** [comment]
I took a look at the provided level but was unable to produce any kind of crash relating to selecting/interacting with paths.
 
Are you using a pre-built CoinKiller binary, or are you building from source? If you are using a pre-built binary, please ensure you are using the latest version.

Could you also provide more information on exactly what you did before the crash occurred?

**Comment by mariomadproductions** [comment]
I built from the latest source. I've uploaded the built in case that helps. It requires Qt6 to be installed in /opt/Qt.
[CoinKiller.zip](https://github.com/Arisotura/CoinKiller/files/6177322/CoinKiller.zip)

To recreate it, I just have to click on the node 2 of path 1 (overlaying the lift), and if that doesn't trigger the crash on its own, then I need to do one of the following:
move the mouse
move the mouse over the the path settings
move the mouse over the path settings and click on them
Then in the console I get
`/home/user/bin/coinkiller: line 4:  6287 Segmentation fault      (core dumped) $folder/CoinKiller`.
`$folder/CoinKiller` is just the command i use in the shell script to launch CoinKiller:
```
#!/usr/bin/env bash
folder=~/opt/CoinKiller
cd $folder
$folder/CoinKiller
```
(maybe CoinKiller can cope with the working directory not being in its folder, so this script may be unneeded)

**Comment by ExplosBlue** [comment]
Was fixed in 611a24db07d565d5f538f3596deb19bfaf988119
